src/pytorch_lightning/overrides/fairscale.py

```diff
@@ -11,27 +11,41 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from typing import Optional, Union
+
 import torch.nn as nn
 
 import pytorch_lightning as pl
-from pytorch_lightning.overrides.base import _LightningModuleWrapperBase, unwrap_lightning_module
-from pytorch_lightning.utilities import _IS_WINDOWS, _module_available
-
-_FAIRSCALE_AVAILABLE = not _IS_WINDOWS and _module_available("fairscale.nn")
```

Comment:
What was the issue with keeping this variable here? https://github.com/Lightning-AI/lightning/issues/11826

Reply:
Oh, I guess I was trying to mitigate a small code duplication. But we will address it anyway later. Reverted.